### DISCUSSION TITLE
i have few issues with pdf generated from Adobe pdf viewer -  1. the pdf generated is pixelated 2. a horizontal white line is appearing

### DISCUSSION THREAD

**Discussion by NiharikaVerma22** [community/community#106565] [Discussions]
### Select Topic Area

Bug

### Body

i have few issues with pdf generated from Adobe pdf viewer - 
1. the pdf generated is pixelated
2. a horizontal white line is appearing

**Comment by queenofcorgis** [comment]
Hi @NiharikaVerma22 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Adobe. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
